backend/prisma/schema.prisma

```diff
@@ -32,8 +54,111 @@ model Session {
 }
 
 model Profile {
-  id     String @id @default(uuid())
-  user   User @relation(fields: [userId], references: [id])
-  userId String @unique @default(uuid())
-  photoUrl String? 
-}
\ No newline at end of file
+  id        String   @id @default(uuid())
+  user      User     @relation(fields: [userID], references: [id])
+  userID    String   @unique
+  photoUrl  String?
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+}
+
+model Plan {
+  id               String       @id @default(uuid())
+  description      String
+  startDate        DateTime
+  endDate          DateTime
+  createdDate      DateTime     @default(now())
+  specialEventType EventType
+  contact          Contact?     @relation(fields: [contactID], references: [id])
+  contactID        String?      @unique
+  User             Pledge[]
+  createdAt        DateTime     @default(now())
+  updatedAt        DateTime     @updatedAt
+  GiftChoice       GiftChoice[]
+}
```

Comment:
Do we need a `createdBy`? 
we also have a `createdAt` and a `createdDate` field. I believe we only need one.

Reply:
For the `createdBy` user, we're storing the user who created the plan in another join table called `Pledge` that joins together the `User` and `Plan` table!

Ok I deleted createdDate, good catch!